dask_geomodeling/geometry/aggregate.py

```diff
@@ -430,15 +428,21 @@ def process(geom_data, raster_data, process_kwargs):
                 if not active.any():
                     continue
 
-                with warnings.catch_warnings():
-                    # we may get divide by 0 if any geometry does not contain
-                    # any 'active' values
-                    warnings.simplefilter("ignore")
-                    agg[frame_no][select] = agg_func(
-                        1 if statistic == "count" else frame[active],
-                        labels=labels[active],
-                        index=index,
-                    )
+                # select features that actually have data
+                # (median and percentile cannot handle it otherwise)
```

Comment:
Actually, the min and max measurements fail the new test as well without this selection, so the comment is not accurate.